compiler/rustc_session/src/options.rs

```diff
@@ -700,6 +702,25 @@ mod parse {
         true
     }
 
+    crate fn parse_cfprotection(slot: &mut CFProtection, v: Option<&str>) -> bool {
+        if v.is_some() {
+            let mut bool_arg = None;
+            if parse_opt_bool(&mut bool_arg, v) {
```

Comment:
If the option accepts `-C cf-protection=y/n/yes/no` then the documentation string should probably tell about that.